clarify rollout behavior in deployment (#28887)
kubernetes/website: content/en/docs/concepts/workloads/controllers/deployment.md

    REVISION  CHANGE-CAUSE
    1   <none>
    ```

> * Get the rollout status to ensure that the existing ReplicaSet is not changed:

Reviewer: ```suggestion
* Get the rollout status to verify that the existing ReplicaSet has not changed:
```
?

Author: Thanks for the rewording, I've updated this sentence.
